app/controllers/catalog_controller.rb

```diff
@@ -221,28 +221,55 @@ def index
     # For comparison of search results with different Solr params:
     # if qt param is passed to Argo, we can also change the params we send to Solr.
     # This allows us to compare, e.g. search results with different Solr qf params.
-    # Request handlers can be wholly configured in Solr, in which case you can remove
-    # default_solr_params to use what is in the Solr configuration.
+    # Request handlers can be configured:
+    #  1. in Solr, in which case you can remove default_solr_params to use only
+    #     what is in the Solr configuration.
+    #  2. in Argo/Blacklight BUT the request handler (qt param) must match an existing request handler in Solr.
+    #  3. configured with url params, e.g. qt=wingnut BUT - if default_solr_params are in play,
+    #     you cannot override those params with url params, and the qt param value has to match an
+    #     existing request handler in Solr.
+    #  4. a combination of the above - some params can be configured in Solr, some in Argo/Blacklight, and some
+    #     in the url.
     if params.key?(:qt)
       blacklight_config.default_solr_params = {
         qt: params[:qt],
         defType: 'dismax',
         'q.alt': '*:*',
         qf: %(
-          id
+          sw_display_title_tesim^5
```

Comment:
This (currently) represents something close to how the qf param in blacklight_config_helper will change soon!